index.html

```diff
@@ -5,51 +5,224 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
-    <title>Portfolio</title>
+    <title>Saikat Portfolio</title>
 </head>
 <body>
-    <div class="main"> 
-        <div class="header">
-            <header>
-                <nav>
-                    <div>
-                        <a href="./index.html">
-                            <img class="logo" src="./image/logo.png" alt="">
-                        </a>
-                    </div>
-                    <div class="navbar">
-                        <div></div>
-                        <div></div>
-                        <div></div>
-                    </div>
-                </nav>
-            </header>
-        </div>
-        <section>
-            <div class="body">
-                <div class="name">
-                    <h1>I’m Saikat</h1>
-                     <h2>Glad to see you!</h2>
+    <div class="container">
+        <div class="main"> 
+            <div class="header">
+                <header>
+                    <nav>
+                        <div>
+                            <a href="./index.html">
+                                <img class="logo" src="./image/logo.png" alt="">
+                            </a>
+                        </div>
+                        <div class="navbar">
+                            <div></div>
+                            <div></div>
+                            <div></div>
+                        </div>
+                    </nav>
+                </header>
+            </div>
+            <section>
+                <div class="body">
+                    <div class="name">
+                        <h2>I’m Saikat</h2>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make only this text an h1 because it is your main title and your webpage needs an h1 for SEO. you can use other headers from `h2` to `h6` for your other headers in your web page

Reply:
changed the same.